vendor/knative.dev/pkg/webhook/env.go

```diff
@@ -22,10 +22,14 @@ import (
 	"strconv"
 )
 
-const portEnvKey = "WEBHOOK_PORT"
+const (
```

Comment:
Have you manually changed this file: vendor/knative.dev/pkg/webhook/env.go?
It is a dependency of this repository. You shouldn't change it here at all.
If you propose a change in knative.dev/pkg, please submit your change in knative.dev/pkg first, then update the dependencies here in this repository.

Reply:
I didn't realized that the pkg repository exist.
I will submit this PR to knative.dev/pkg first.

Thanks.